import           System.Exit (exitFailure)

main :: IO ()
main = exitFailure
